R/osmapiR_connection.R

```diff
@@ -158,9 +158,18 @@ authenticate_osmapi <- function() {
 #' @rdname authenticate_osmapiR
 #' @export
 logout_osmapi <- function() {
-  req <- httr2::request(base_url = get_osmapi_url())
-  req <- oauth_request(req)
-  req$policies$auth_oauth$cache$clear()
+
+  if (utils::packageVersion("httr2") >= "1.0.5.9000") {
```

Comment:
```suggestion
  if (utils::packageVersion("httr2") >= "1.0.6") {
```
Looking the recent history of httr2 that's the version we can rely on `oauth_cache_clear()`.

Reply:
Now that it's released, you could just declare `(>= 1.0.6)` in the `DESCRIPTION` and drop all the optional code.